packages/components/organisms/f-checkout/src/demo/get-basket-collection.json

```diff
@@ -1,4 +1,5 @@
 {
+  "BasketId": "11111111",
```

Comment:
Maybe a silly question, but is this related to the PR or is it something else? 😄

Reply:
I think it's adding in bits from another PR. But if they're not there it's harder to test the analytics stuff